mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a,b,c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
```

Comment:
That is right, after writing your function definition, you should always surround the body of your function with curly brackets, otherwise, JavaScript won't understand that the body you wrote belongs to that function.

I also liked how you used String interpolation in this part.